src/tlv/TlvObject.ts

```diff
@@ -97,8 +107,8 @@ export const TlvObject = <F extends TlvFields>(fields: F) => new ObjectSchema(fi
 /** List TLV schema. */
 export const TlvList = <F extends TlvFields>(fields: F) => new ObjectSchema(fields, TlvType.List);
 
-/** Object TLV mandatory field. */
-export const TlvField = <T>(id: number, schema: TlvSchema<T>) => ({ id, schema, optional: false }) as FieldType<T>;
+/** Object TLV mandatory field. A fallback value can be defined for backward compatibility.  */
```

Comment:
Isn't `defaultValue` a better, more self-explanatory term than `fallback`?

Reply:
In this case it is formally not a default value "as per specs" (and we use this term "default" already for Default values of attributes in the clusters). Sp from my POV "fallback" is fine because it is the "if the value was discarded while sending it is easier for our code to use this fallback value" :-)

Reply:
Would it be possible to expand the comment to explain better to a layperson how it is used by the API to inform the caller how to set it with intention?

Reply:
my try see https://github.com/project-chip/matter.js/pull/50/files#r1103324471